linkingFuncs.py

```diff
@@ -25,6 +25,62 @@ def get_id(username, cur):
         return None
     return fetch[0]
 
+@app.route("/buy/<username>/<tik>/<volume>")
+def buy(username, tik, volume):
+    # establish connection
+    cnx = mysql.connector.connect(user='root', password='Valentino46', database='StonkLabs')
+    # create a cursor 
+    cur = cnx.cursor(buffered=True)
+    # get current price
+    price =  search_tiker(tik).get('price')
+    if(price == {}):
+        return { 'Action': False}
+    id = get_id(username, cur)
+    if (id ==  None):
+        return { 'Action': False}
+    time = datetime.datetime.now()
+    cur.execute("INSERT INTO Transactions Values ('%d', '%s', '%d', 'BUY', '%f', '%s', 0)" % (
+        id, tik, volume, price, time))
+    # Commit change 
+    cnx.commit()
+    # Close connections 
+    cnx.close()
+    return { 'Action': True}
+
+@app.route("/sell/<username>/<tik>/<volume>")
+def sell(username, tik, volume):
+        # establish connection
+    cnx = mysql.connector.connect(user='root', password='Valentino46', database='StonkLabs')
+    # create a cursor 
+    cur = cnx.cursor(buffered=True)
+    # get current price
+    price =  search_tiker(tik).get('price')
+    if(price == {}):
+        return { 'Action': False}
+    id = get_id(username, cur)
+    if (id ==  None):
+        return { 'Action': False}   
+    time = datetime.datetime.now()
+    cur.execute("SELECT * FROM Transactions WHERE ticker='%s' AND UserId='%d'" % (tik, id))
+    curVolume = 0
+    fetch = cur.fetchall()
+    if (fetch == []):
+        return { 'Action': False }
+    for item in fetch:
```

Comment:
Could be nice to add a comment describing the contents of item so that the indexing is a bit more clear

Reply:
Global "label" variables will likely be added soon to indicate the labels